src/network/http.js

```diff
@@ -0,0 +1,30 @@
+class http {
+  constructor(config) {
+    this.baseUrl = `http://${config.hostname}:${config.port}`;
+    this.header = {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    };
+  }
+
+  async post(path, body) {
+    const url = `${this.baseUrl}/${path}`;
+
+    const options = {
+      method: 'POST',
+      headers: this.header,
+      body: body && JSON.stringify(body),
+    };
```

Comment:
Not sure if this is correct, but it should be something along these lines now :point_down: 

```suggestion
  async post(path, body, authToken) {
    const url = `${this.baseUrl}/${path}`;

    const options = {
      method: 'POST',
      headers: {
        Accept: 'application/json',
        'Content-Type': 'application/json',
        Authorization: authToken,
      },
      body: body && JSON.stringify(body),
    };
```